frontend/src/index.css

```diff
@@ -155,6 +161,16 @@ button {
 .ig-post-padding {
   padding: 20px;
 }
+.BackButton {
+  width: 40px;
+  height: 40px;
+  border: solid #000000 2px;
+}
+.BackButtonImg {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
```

Comment:
Considering using [CSS Modules](https://github.com/css-modules/css-modules). When the project scales and you have a global CSS file defined, weird issues can happen, if you have defined the same class by mistake twice.

Reply:
I will have a look into it and maybe implement it later on.